ecosystem/sep-0010.md

```diff
@@ -24,6 +26,7 @@ It involves the following components:
     - The `SIGNING_KEY` from the **Home Domain** is the **Server Account**.
 - A **Client**: the software used by the holder of the account being authenticated by the **Server**.
     - The account being authenticated is the **Client Account**.
+    - A Stellar account (`G...`) or a muxed account (`M...`).
```

Comment:
💡 I think this few lines could do with a little tweaking. In the sub-points we discuss the account mostly, not the client, so I think the Client Account deserves it's own top-level point.

I notice we're also missing G+memo from the list of possible values of the Client Account. I think that's important because a G+memo is not authenticating the G alone.

```suggestion
- A **Client Account**: the account being authenticated.
    - A Stellar account (`G...`) or a muxed account (`M...`).
    - If a Stellar account (`G...`), may be accompanied by a memo to scope the authentication to a user or sub-account of the account.
- A **Client**: the software used by the holder of the **Client Account** being authenticated by the **Server**.
```